Packs/MySQLEnterprise/ModelingRules/MySQLEnterpriseEventCollector/MySQLEnterpriseEventCollector.xif

```diff
@@ -0,0 +1,28 @@
+[MODEL: dataset=mysql_enterprise_raw, model=Audit, content_id=***, version=***]
+| alter XDM.Audit.operation.type = json_extract_scalar(_raw_log, ""$.class"")
+| filter XDM.Audit.operation.type = ""audit""
+| alter XDM.Database.time = json_extract_scalar(_raw_log, ""$.timestamp""),
```

Comment:
I think this is probably an error and it's supposed to be `XDM.Audit.event_timestamp`...

Reply:
Yes